BaseFilter/sections/specific.txt

```diff
@@ -6870,6 +6870,7 @@ tmonews.com##.sponsor-dfp
 fc.lc###link-view > iframe[src*="/fc.lc/"]
 news.sky.com##.sdc-site-au[data-ad-format] > div[data-role="ad-container"]
 news.sky.com##.sdc-site-layout-sticky-region__ghost[data-format="leaderboard"]
+!+ NOT_OPTIMIZED
```

Comment:
hints must be at the and of rules group or a section.
In this case - here
https://github.com/AdguardTeam/AdguardFilters/blob/f507bd5ba6f5c8472a7f77b85ecd33b3b4972318/BaseFilter/sections/specific.txt#L13889

Reply:
Moved.